pkg/controller/cassandracluster/generator.go

```diff
@@ -577,33 +585,15 @@ func createEnvVarForCassandraContainer(cc *api.CassandraCluster, status *api.Cas
 				},
 			},
 		},
-		v1.EnvVar{
-			Name: "POD_NAME",
-			ValueFrom: &v1.EnvVarSource{
-				FieldRef: &v1.ObjectFieldSelector{
-					APIVersion: "v1",
-					FieldPath:  "metadata.name",
-				},
-			},
-		},
-		v1.EnvVar{
+		{
 			Name:  "CASSANDRA_GC_STDOUT",
 			Value: strconv.FormatBool(cc.Spec.GCStdout),
 		},
-		v1.EnvVar{
+		{
 			Name:  "CASSANDRA_NUM_TOKENS",
 			Value: strconv.Itoa(int(numTokensPerRacks)),
 		},
-		v1.EnvVar{
-			Name: "NODE_NAME",
```

Comment:
Did you remove this one because it's unused ?

Reply:
yes